delphi/translators/for2py/floatNumpy.py

```diff
@@ -0,0 +1,149 @@
+from numpy import float32
+
+################################################################################
+class FloatNumpy:
+    def __init__(self, val):
+        self._val = float32(val)
+
+    def __add__(self, other):
+        if isinstance(other, FloatNumpy):
```

Comment:
This if statement is repeated in every method of this class.  Such repetition of code is undesirable.  Instead, I suggest defining a method like the following:
```
def get_value(self, other):
    if isinstance(other, Float32):
        return other._val
    else:
        return other
```
Now you can define `__add__()` as:
```
def __add__(self, other):
    return self._val + self.get_value(other)
```
The other operators are analogous.

More generally, we should try to avoid repeating code.  That's bad style.  (I know you inherited this code from me, so the bad style here is originally my fault.  But it still needs to be fixed.)